tests/recipe/test_amplitude_creation.py

```diff
@@ -33,7 +33,17 @@ def test_helicity(helicity_model: es.AmplitudeModel):
 
     builder = IntensityBuilder(model.particles, kinematics, phsp_sample)
     intensity = builder.create_intensity(model)
-    assert len(intensity.parameters) == 9
+    assert set(intensity.parameters) == {
+        "strength_incoherent",
+        "Position_J/psi(1S)",
+        "Width_J/psi(1S)",
```

Comment:
Is it intended that the J/psi has a BreitWigner dynamics?

Reply:
I wouldn't say intended, but currently, mass and width are always registered, no matter what the dynamics are...
https://github.com/ComPWA/tensorwaves/blob/d2d1b90a26b532bc16a132f538a43153e2e87426/src%2Ftensorwaves%2Fphysics%2Fhelicity_formalism%2Famplitude.py#L591-L632

I would say this should be properly tackled in e.g. #162. This PR just removes the default dynamics (particularly the buggy way in which it was implemented).